tutorials/sar-basics/index.md

```diff
@@ -0,0 +1,181 @@
+
+// Check which sensor/relative orbit combinations we have
+var orbits = ee.Dictionary(s1.aggregate_histogram('platform_relorbit'))
+
+print(orbits) // Check in console
+
+var keys = orbits.getInfo() // Needed to loop
+
+for (var k in keys) {
+  var color = 'blue'
+  if (k.charAt(0)==='B') {
+    color = 'green'
+  }
+  Map.addLayer(ee.Image().paint(ee.FeatureCollection(s1).filterMetadata('platform_relorbit', 'equals', k), 0, 1), {palette: [color]}, "Footprint" + k, false);
+}
+
+for (var k in keys) {
+  Map.addLayer(s1.filterMetadata('platform_relorbit', 'equals', k).first(), {bands: ['angle'], min: 30, max: 45 }, "Incidence angle: " + k, false);
+}
+
+Map.addLayer(ee.Image().paint(geometry, 0, 1), {palette: ['red']}, 'AOI', false);
+          
+```
+
+If you run the script *as is* you'll notice that the, relatively small, area of interest is where 2 DESC orbits overlap (with relative orbit numbers 37 and 110) AND also 2 ASC orbits (15 and 88). For both Sentinel-1A and -1B combined, that makes for 8 distinct coverages in a full orbit cycle. By displaying the individual footprints of each of the scenes, you should note that DESC scenes are indeed rotated slightly to the North-East, and ASC scenes to the North-West. 
+
+A curiosity is the situation for Sentinel-1B orbit 88. The scene boundary cuts right through the AOI. This is linked to the 25 seconds azimuth slice limits. Unfortunately, the azimuth slicing is not synchronized between Sentinel-1A and -1B (and may even drift somewhat over time). This means you need to compose the full AOI image cover by compositing two adjacent scenes in this case.
+
+Now switch on the "Incidence Angle" layers. These layers are generated from the "angle" band of each S1_GRD scene. Verify with the "Inspector" that incidence angle varies in the range direction between about 30-39 degrees. Since we just learned that the lowest incidence angle is in the near range, we can confirm that Sentinel-1 is a right looking SAR. 
+
+Finally, inspect some of the VV and VH values for points inside the AOI. Switch on the Google satellite background to select specific land use classes (e.g. urban area, grassland, arable crops). Since we're using the S1_GRD catalog, values are expressed in decibels (dB), i.e. on a logarithmic scale. Point samples show significant variation for the different acquisitions. This is expected, because the diversity in orbit look and incidence angles is wide for this AOI, we have not accounted for speckle effects and neither for the environmental factors (weather, land preparation, crop growth) that influence Sentinel-1 backscatter. The good news is that we have good dynamics in the VV and VH backscattering signal in space and time. The downside is that we now need to control for the various sensor and environmental parameters to start to make sense out of this observations. At this point, mono-dimensional NDVI huggers have probably switched off, if they got this far in the first place...
```

Comment:
"mono-dimensional NDVI huggers have probably switched off, if they got this far in the first place..."

I don't mind a bit of snark here, but maybe we could tone it down a bit?

Reply:
removed